source/common/redis/command_splitter_impl.h

```diff
@@ -128,8 +128,97 @@ struct Commands {
   static const std::vector<std::string>& allToOneCommands() {
     // TODO(danielhochman): DEL should hash to multiple servers and is only added for testing single
     // key deletes
-    static const std::vector<std::string>* commands = new std::vector<std::string>{
-        "decr", "decrby", "del", "expire", "get", "getset", "incr", "incrby", "set", "setex"};
+    static const std::vector<std::string>* commands =
```

Comment:
nit: Can we potentially move this to a new file called supported_commands.h/cc or something inside of a utility class? It would just be nice to not have this file become giant and strange to look at due to the formatting rules.